### PR TITLE
Footer Submit button UI update

### PR DESCRIPTION
**Pull request by riya123-hub**
Changes made in the footer submit button in the projects.html file

### CODE REVIEW

File: assets/css/project.css [hunk 1]
```diff
@@ -1,3 +1,4 @@
+
 html[data-theme = 'light']
```
Comment (smaranjitghose) [inline assets/css/project.css:2]: Remove this additional line.

Git ignore the .DS_Store file

**Review by smaranjitghose — CHANGES_REQUESTED** [review]

**Comment by smaranjitghose** [comment]
The button itself looks cool but does not go well with the rest of the theme.

@anushbhatia  could you please give some suggestions for the same

**Comment by riya123-hub** [comment]
> The button itself looks cool but does not go well with the rest of the theme.
> 
> * Please follow the first reference given in the corresponding issue
> * The code should be well documented (Add comments wherever necessary)
> * The PR description should mention the issue number as well as a snapshot/clip of the demo after your changes

Okay I will try to make the requested updates done

**Review by deeqakkk — COMMENTED** [review]
okay, it's fine @riya123-hub

**Comment by smaranjitghose** [comment]
Closed as the PR is not as per the requirement even post a review.

**Pull request closed without merging**